src/mpi/init/mpir_init.c

```diff
@@ -55,12 +55,9 @@ extern const char MPII_Version_device[];
 /* MPIR_world_model_state tracks so we only init and finalize once in world model */
 MPL_atomic_int_t MPIR_world_model_state = MPL_ATOMIC_INT_T_INITIALIZER(0);
 
-/* Use init_lock to protect concurrent init/finalize (include session init/finalize) */
-static MPL_initlock_t init_lock = MPL_INITLOCK_INITIALIZER;
-
```

Comment:
I was initially confused by this commit because I expected to see additional locking added. But it seems this is just renaming/exposing this lock for use in future commits. No change necessary, in the code. Maybe an update to the commit message for clarity.

Reply:
Commit message updated.